bazel/extension_config/extensions_build_config.bzl

```diff
@@ -486,7 +486,7 @@ ISTIO_ENABLED_CONTRIB_EXTENSIONS = [
     "envoy.filters.sip.router",
     "envoy.tls.key_providers.cryptomb",
     "envoy.tls.key_providers.qat",
-    "envoy.network.connection_balance.dlb",
+#    "envoy.network.connection_balance.dlb",
```

Comment:
`select` cannot work here, @kyessenov any suggestion to make this filter optional on macos?

Reply:
https://github.com/envoyproxy/envoy/pull/30951 may give some help.

Reply:
Yeah, you have to fix it upstream to guard the code on macos.

Reply:
https://github.com/envoyproxy/envoy/pull/30951 merged